src/mds/MDLog.h

```diff
@@ -293,7 +294,7 @@ class MDLog {
   void try_to_commit_open_file_table(uint64_t last_seq);
   LogSegment* _start_new_segment(SegmentBoundary* sb);
   void _segment_upkeep();
-  void _submit_entry(LogEvent* e, MDSLogContextBase* c);
+  LogSegment::seq_t _submit_entry(LogEvent* e, MDSLogContextBase* c);
```

Comment:
To avoid this in the future, we should pass a smart unique_ptr (r-value) to the LogEvent.